frontend/src/components/admin/AdminDashboard.js

```diff
@@ -27,30 +56,104 @@ export class AdminDashboard extends Component {
     downloadMineManagerHistory(this.state.mineNo, { window, document });
   };
 
```

Comment:
Is this being sorted alphabetically? Looks like there is a JS function that does this, `localeCompare()`

If that's not the purpose of this function, can you rename to make it more clear whats happening?